src/ecs/explorer/ecsClusterNode.ts

```diff
@@ -23,6 +23,7 @@ import { getLogger } from '../../shared/logger'
  */
 export class EcsClusterNode extends AWSTreeNodeBase implements AWSResourceNode, LoadMoreNode {
     private readonly childLoader: ChildNodeLoader
+    public persistChildren: boolean = false
```

Comment:
this code is copy-pasted in 5+ classes. why not put it in `AWSTreeNodeBase` instead?